staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -217,7 +217,7 @@ func (flags *ApplyFlags) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVar(&flags.Overwrite, "overwrite", flags.Overwrite, "Automatically resolve conflicts between the modified and live configuration by using values from the modified configuration")
 	cmd.Flags().BoolVar(&flags.Prune, "prune", flags.Prune, "Automatically delete resource objects, that do not appear in the configs and are created by either apply or create --save-config. Should be used with either -l or --all.")
-	cmd.Flags().StringVarP(&flags.Selector, "selector", "l", flags.Selector, "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
+	cmd.Flags().StringVarP(&flags.Selector, "selector", "l", flags.Selector, "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2). Matching objects must satisfy all of the specified label constraints.")
```

Comment:
@ncapps There are more commands (for example `kubectl get`) that has `label` flag, I think it would make sense to update all the command that has the flags label

Reply:
Thank you for the feedback. The helper message is updated as requested.